xml/System.Runtime.InteropServices/Marshal.xml

```diff
@@ -7632,7 +7632,7 @@ On .NET 6 and later versions, this method is functionally equivalent to <xref:Sy
 
 ## Remarks
  <xref:System.Runtime.InteropServices.Marshal.PtrToStructure%60%601%28System.IntPtr%29> is often necessary in COM interop and platform invoke when structure parameters are represented as <xref:System.IntPtr?displayProperty=nameWithType> values. You can pass a value type to this method overload.
- If the `ptr` parameter equals <xref:System.IntPtr.Zero?displayProperty=nameWithType>, `null` will be returned.
+ If the `ptr` parameter equals <xref:System.IntPtr.Zero?displayProperty=nameWithType> and `T` is not a value type, `null` will be returned. Otherwise, a <xref:System.NullReferenceException> will be thrown.
```

Comment:
Does the "otherwise" mean `T` is a value type and `ptr` is IntPtr.Zero?

Reply:
Yes.

> If `T` is a value type and `ptr` is `IntPtr.Zero`, then a `NullReferenceException` will be thrown.

Would the above sentence be more appropriate?